src/blockchain/Blockchain.spec.ts

```diff
@@ -5,6 +5,26 @@ import { createShardAccount, GetMethodError, TimeError } from "./SmartContract";
 import { internal } from "../utils/message";
 import { SandboxContractProvider } from "./BlockchainContractProvider";
 import { TickOrTock } from "../executor/Executor";
+import { runTolkCompiler } from "@ton/tolk-js";
+
+async function compileTolk(source: string) {
```

Comment:
lets extract this to different file. I think introducing folder like __test_utils__ would work nice